src/modules/ethereum/chains/filterNetworkByQuery.ts

```diff
@@ -12,7 +12,7 @@ function contains(str1?: string, str2?: string) {
 export function filterNetworksByQuery(query: string) {
   return (item: NetworkConfig) =>
     contains(item.id, query) ||
-    contains(Networks.getChainId(item) || '', query) ||
+    (Networks.isEip155(item) && contains(Networks.getChainId(item), query)) ||
```

Comment:
should we add solana support to this filter?

Reply:
What do you mean? Solana `NetworkConfig` is the same, only `chainId` can't be inferred